File: ce/outlook-app/snippets/csharp/CRMV8/query/cs/linqexamples15.cs
// <snippetlinqexamples15>


 using (ServiceContext svcContext = new ServiceContext(_serviceProxy))
 {
  var query_equals2 = from c in svcContext.ContactSet
                      where c.FamilyStatusCode.Equals(3)
                      select new
                      {
                       c.FirstName,
                       c.LastName,
                       c.Address1_City
                      };
  foreach (var c in query_equals2)
  {
   System.Console.WriteLine(c.FirstName +
    " " + c.LastName +
    " " + c.Address1_City);
  }
 }

// </snippetlinqexamples15>

File: ce/customerengagement/on-premises/snippets/visualbasic/CRMV8/queryvb/vb/linqexamples15.vb
' <snippetlinqexamples15>


 Using svcContext As New ServiceContext(_serviceProxy)
  Dim query_equals2 = From c In svcContext.ContactSet _
                      Where c.FamilyStatusCode.Equals(3) _
                      Select New With {Key c.FirstName,
                                       Key c.LastName,
                                       Key c.Address1_City}
  For Each c In query_equals2
   Console.WriteLine(c.FirstName &amp; " " &amp; c.LastName &amp; " " _
                     &amp; c.Address1_City)
  Next c
 End Using

' </snippetlinqexamples15>